docs/ConfigureScalarDL.md

```diff
@@ -18,8 +18,14 @@ You need to create a secret object with `db-username` and `db-password` to store
     # username=<AWS_ACCESS_KEY_ID>
     # password=<AWS_ACCESS_SECRET_KEY>
 
+# AWS RDS Databases(MySQL PostgreSQL, Oracle, SQL Server, Aurora), Azure Databases for MySQL, PostgreSQL
+   # username=<ACCOUNT_USERNAME>
+   # password=<ACCOUNT_PASSWORD>
+    
 kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
 ```
+**Note**:- 
+* For Azure Database for MySQL use <ACCOUNT_USERNAME@HOST_NAME> instead of <ACCOUNT_USERNAME> as `username` value.
```

Comment:
Ditto

Reply:
Removed the notes section for now as it may be a bit confusing.

Reply:
I think we can keep this note if it is needed.